src/fides/cli/options.py

```diff
@@ -40,25 +44,22 @@ def fides_key_option(command: Callable) -> Callable:
         "-k",
         "--fides-key",
         default="",
-        help="The fides_key of the single policy that you wish to evaluate.",
     )(command)
     return command
 
 
 def manifests_dir_argument(command: Callable) -> Callable:
     "Add the manifests_dir argument."
     command = click.argument(
-        "manifests_dir",
-        default=".fides/",
-        type=click.Path(exists=True),
+        "manifests_dir", type=click.Path(exists=True), default=".fides/"
     )(command)
     return command
 
 
 def dry_flag(command: Callable) -> Callable:
     "Add a flag that prevents side-effects."
     command = click.option(
-        "--dry", is_flag=True, help="Prevent the persistance of any changes."
+        "--dry", is_flag=True, help="Do not upload results to the Fides webserver."
```

Comment:
Looks like there's other places this dry flag could be used?

Reply:
I tried to use specific instances of this flag elsewhere when a different `help` message made more sense